ot_filter.go

```diff
@@ -0,0 +1,50 @@
+package otrestful
+
+import (
+	"github.com/emicklei/go-restful"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+)
+
+// go-restful FilterFunction does not allow to opt in third-party parameters,
+// so using global variables for the moment
+// TODO(mqliang): using parameter instead of global variable to set this
+var (
+	Tracer            opentracing.Tracer
+	OperationNameFunc func(r *restful.Request) string
+	ComponentName     string
+)
+
+var (
+	DefaulOperationNameFunc = func(r *restful.Request) string {
+		return "HTTP " + r.Request.Method
```

Comment:
Why the `operationName` is `"HTTP"` + `"Request.Mehod"`? I am not a go-developer, the `"Request.Mehod"` seems to be `GET`, `POST` etc.

IMO, the operation name should be more likely `request.uri`.